src/components/purpose-card.js

```diff
@@ -75,10 +77,76 @@ class PurposeCard extends React.Component {
     })
   }
 
+    handleEditorChange = (content) => {
+    const { activeGroup, field } = this.props
+    const { data } = activeGroup
+
+    this.setState({[field]: content.level.content})
+
+  }
+
+  rtffield = () => {
```

Comment:
Instead of doing this, I was suggesting you to keep this in a separate file. And then you can simply import it as `import RTField from ../fields/rt-field`.